test/unit/default_appearance_spec.js

```diff
@@ -50,4 +52,148 @@ describe("Default appearance", function () {
       });
     });
   });
+
+  describe("parseAppearanceStream", () => {
+    it("should parse a FreeText (from Acrobat) appearance", () => {
+      const appearance = `
```

Comment:
Here, and in the tests below, I'd suggest defining this using the "correct" format since that makes it clearer what the correct data is.
```suggestion
      const appearance = new StringStream(`
```